OWML.ModLoader/Owo.cs

```diff
@@ -38,8 +44,28 @@ public void LoadMods()
                 Application.logMessageReceived += OnLogMessageReceived;
             }
             var mods = _modFinder.GetMods();
-            foreach (var modData in mods)
+
+            var sortedMods = _sorter.SortMods(mods);
+
+            var modNames = new List<string>();
+            foreach (var mod in mods)
+            {
+                if (mod.Config.Enabled == true)
+                {
+                    modNames.Add(mod.Manifest.Name);
+                }
+            }
+
+            foreach (var modDep in sortedMods)
             {
+                foreach (var dep in modDep.Dependencies)
+                {
+                    if (!modNames.Contains(dep) && dep != "None")
+                    {
+                        _console.WriteLine("Error! " + modDep.Name + " needs " + dep + ",  but it's disabled!");
```

Comment:
You can use string interpolation:
_console.WriteLine($"Error! {modDep.Name} needs {dep},  but it's disabled!");